clerk/tokens_test.go

```diff
@@ -235,6 +235,18 @@ func TestClient_VerifyToken_Success_AuthorizedParty(t *testing.T) {
 	}
 }
 
+func TestClient_VerifyToken_Success_WithJWTVerificationKey(t *testing.T) {
+	c, _ := NewClient("token")
+	token, pubKey := testGenerateTokenJWT(t, dummySessionClaims, "kid")
+
+	// How to convert public key to a string?
+	// The conversion below is fake and will fail.
+	_, err := c.VerifyToken(token, WithJWTVerificationKey(pubKey.(string)))
```

Comment:
@chanioxaris I could use some help here. 😊 For this test to work, I need to convert the public key (object) to a verification key (string). Unfortunately, I haven't found a way to do that. Maybe you have some idea.